redwine/views.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 import datetime, re
+import pytz
```

Comment:
https://docs.djangoproject.com/en/4.2/releases/4.0/#zoneinfo-default-timezone-implementation

Can this be change to not use pytz? I have a branch for OW4 that removes direct usages of pytz as that dependency is no longer needed (`zoneinfo` from the standard library in python 3.9 replaces it)

`pytz` is also a separate package that is not in this project's `requirements.txt`, not that is up-to-date in any way

Reply:
That can certainly be fixed. I probably should have added pytz to `requirements.txt` even though it's a direct dependency of django. I'll change it out for the zoneinfo standard library either way though :)